src/models/methods/account_offers.ts

```diff
@@ -0,0 +1,32 @@
+import { Amount, LedgerIndex } from "../common";
+import { BaseRequest, BaseResponse } from "./base_method";
+
+export interface AccountOffersRequest extends BaseRequest {
```

Comment:
looks like you branched off of mayukha and she hasn't merged yet - so we have a lot of noise in this PR

Reply:
Yeah, he needs to re-rebase off of my branch